pyproject.toml

```diff
@@ -639,3 +648,51 @@ module = [
 ]
 disallow_untyped_defs = true
 # end: stronger typing
+
+# beginning: modules with possibly undefined variables
+[[tool.mypy.overrides]]
+module = [
```

Comment:
unless you're going to make an autofixer for this we shouldn't do this.  let's fix the issues first and then enable the error code

Reply:
yeah I mean more something which whittles this allowlist down automatically (how the rest of the allowlist works) -- but I think we should fix the problems first anyway

Reply:
ah i see what you're syaing